### PR TITLE
[doc] Fix broken links in troubleshooting doc

### PR DESCRIPTION
**Pull request by anik120**
Some link addresses were changed due to PR #176 but the links
were not updated. This PR fixes the broken links with the
correct link addresses.

### CODE REVIEW

**Comment by anik120** [comment]
@aravindhp added space.

File: docs/troubleshooting.md [hunk 1]
```diff
@@ -34,7 +34,7 @@ redhat-operators      appregistry   https://quay.io/cnr   redhat-operators
 ```
 
 If the status of any of the OperatorSource is anything other than `Succeeded`, you can investigate the OperatorSource with 
-`oc describe opsrc <Name-of-OperatorSource> -n openshift-marketplace`. [This section](#operatorSource-failing-to-download-from-datastore) of the doc also contains helpful information about debugging an OperatorSource. 
+`oc describe opsrc <Name-of-OperatorSource> -n openshift-marketplace`. [This section](#operatorSource-failing-to-connect-to-the-datastore) and [the following section](#operator(s)-in-an-operatorsource-fail-to-show-up-in-the-ui) of the doc also contains helpful information about debugging an OperatorSource.
```
Comment (galletti94) [inline docs/troubleshooting.md]: the `the following section` link doesn't work

Comment (galletti94) [reply docs/troubleshooting.md]: See https://github.com/operator-framework/operator-marketplace/blob/b1bf31056e8a6dec8c468d2fbc3e1477df65e1ee/docs/troubleshooting.md#operator(s)-in-an-operatorsource-fail-to-show-up-in-the-ui for reference

Comment (anik120) [reply docs/troubleshooting.md]: That is weird. Seems to be working in VS code preview. Let me see what's going on. Thanks for noticing it @galletti94

Comment (anik120) [reply docs/troubleshooting.md]: @galletti94 fixed in the latest push. Please review.

**Comment by kevinrizza** [comment]
These links:

https://github.com/operator-framework/operator-marketplace/blob/92847f5a49230333ba8ad9443987b7d5865d055c/docs/troubleshooting.md#no-packages-show-up-in-the-ui-(no-operatorhub-items-found)

and

https://github.com/operator-framework/operator-marketplace/blob/92847f5a49230333ba8ad9443987b7d5865d055c/docs/troubleshooting.md#operatorsource-%60message%60-reads-unknown-error-(status-404)

Still don't work.

**Comment by anik120** [comment]
@aravindhp @kevinrizza @galletti94 fixed all links. Please review.

**Pull request merged**